### PR TITLE
[Website] Add breadcrumbList, close #143

### LINKED ISSUE #143: BreadcrumbList support
- http://schema.org/BreadcrumbList
- https://developers.google.com/search/docs/data-types/breadcrumbs

### PR DESCRIPTION
**Pull request by qas612820704**
#143 
screenshot from 2017-05-11 01-21-14

How about this? :)

### CODE REVIEW

**Comment by qas612820704** [comment]
I just place it to `/` and `/libraries/`.

It's not available in like the page `/libraries/XXXX` or others now,
because I don't have good idea where to place the breadcrumbList.

screenshot from 2017-05-11 01-21-14

Any suggestion? :)

**Comment by PeterDaveHello** [comment]
@qas612820704 sorry for the late review, would you mind to fix the conflicts first? Thanks.

**Comment by qas612820704** [comment]
I rebased it to latest master branch.
Is conflict resolved?

**Comment by PeterDaveHello** [comment]
Looks so! @qas612820704 for library, I think you can just put it above the library name.
We also need to add it to `about` & `api` page :)

**Comment by qas612820704** [comment]
`library page`
screenshot from 2017-07-02 21-35-54

---

`api page`
screenshot from 2017-07-02 21-36-08

---

`about page`
screenshot from 2017-07-02 21-35-39

**Comment by qas612820704** [comment]
@PeterDaveHello,

While I adding features,

 I saw that `templates/api.html` indents  by **tab** unlike other pages use **space**.

Should I fix this? (Replace tabs to spaces)

**Comment by PeterDaveHello** [comment]
@qas612820704 Nice catch! Feel free to send another PR to fix it, thanks!

**Comment by qas612820704** [comment]
OKey : )

**Comment by PeterDaveHello** [comment]
@PeterDaveHello one more thing, mind to add this feature also on each library version?

File: templates/about.html [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+        </li>
+      {{/breadcrumbList}}
+    </ol>
+  </div>
+  </div>
+  <div class="row">
   <div class="col-md-6">
     <p style="margin-top: 6px;">
-    
+
```
Comment (PeterDaveHello) [inline templates/about.html]: This line may not be related.

Comment (qas612820704) [inline templates/about.html]: Oh, there has space originally, but atom auto deleted it. I will modify back.

File: templates/api.html [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
 	<div class="row">
 		<div class="col-md-12">
+	    <ol class="mybreadcrumb text-muted">
+	      {{#breadcrumbList}}
+	        <li>
+	          {{#last}}<span>{{index}}</span>{{/last}}
+	          {{^last}}<a href="{{url}}">{{index}}</a>{{/last}}
+	        </li>
+	      {{/breadcrumbList}}
+	    </ol>
```
Comment (PeterDaveHello) [inline templates/api.html]: Maybe we should give these 8 lines more spaces as indent?

**Comment by PeterDaveHello** [comment]
Hi @qas612820704 

Can we have more space between the breadcrumbList and the library name?
default

I also wonder if we can hide the "home" item or put it another space? (no idea yet)
default

What do you think?

**Comment by qas612820704** [comment]
@PeterDaveHello, conflict fixed!

**Comment by MattIPv4** [comment]
Closing as duplicate.

**Pull request closed without merging**